yarn.lock

```diff
@@ -695,10 +695,10 @@
   resolved "https://registry.yarnpkg.com/@justeat/f-utils/-/f-utils-0.2.0.tgz#b2333b5ddf20b4157af25091e6bc54706358100c"
   integrity sha512-lmnjGTAburU/rNsylj4ScWqApL/VifiwTce+hfujOJBZHHMUnJ/tmuVy3lxFtxtZIzCMTb502sB6ZjZbH82SOA==
 
-"@justeat/fozzie-colour-palette@2.3.0":
-  version "2.3.0"
-  resolved "https://registry.yarnpkg.com/@justeat/fozzie-colour-palette/-/fozzie-colour-palette-2.3.0.tgz#1ad4b7e827b6c7c7bb0a73d2b72cf8d608fc09b5"
-  integrity sha512-eVzfPhKojCQ0IxXQQL73fnami/ffjtwgRIcnnFik9E/GmJ26WtoxFrBCL6YksmEKU8JKP9tqVkLsC+JqYbjB9Q==
+"@justeat/fozzie-colour-palette@^2.5.0":
+  version "2.5.0"
+  resolved "http://packages.je-labs.com/npm/private-npm/@justeat/fozzie-colour-palette/-/fozzie-colour-palette-2.5.0.tgz#a26555e476c6103cdcd9fdf233238af6b0321cee"
```

Comment:
Probably shouldn't reference internal resources in an external public repo?

Reply:
Good catch 👍